src/_posts/databases/2000-01-01-maintenance-windows.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Database Maintenance Windows
+nav: Maintenance Windows
+modified_at: 2024-11-28 12:00:00
+tags: databases maintenance
+index: 5
+---
+
+
+## Understanding Database Maintenance Windows
+
+**Database Maintenance Windows** is Scalingo's system for scheduling
+*[maintenance operations](#understanding-maintenance-operations)* on your
+databases. On the customer's side, it mainly consists in an 8 hours timespan
+during which database maintenance operations can be scheduled. The goal of
+maintenance windows is to minimize the impact of maintenance operations on your
+application, while facilitating continuous improvement on our side.
+
+{% note %}
+Currently, Scalingo only leverages maintenance windows to force
+[major-upgrades]({% post_url databases/postgresql/2000-01-01-upgrading %})
+of databases when very specific conditions are met, such as an EOL version
+still running, or critical security issues.
+{% endnote %}
```

Comment:
question: are you sure about this? I don't think it's the only reason we want to use maintenance windows. I think we also want this for minor upgrades or things like that.

Reply:
I chose to remove the note, as we already explain what Maintenance Operations are.
Seems way more future-proof like this. Thanks!